micrometer-core/src/main/java/io/micrometer/core/instrument/binder/grpc/GrpcObservation.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.grpc;
+
+import io.micrometer.common.docs.KeyName;
+import io.micrometer.observation.Observation.Context;
+import io.micrometer.observation.Observation.Event;
+import io.micrometer.observation.ObservationConvention;
+import io.micrometer.observation.docs.ObservationDocumentation;
+
+/**
+ * {@link ObservationDocumentation} for gRPC.
+ *
+ * @author Tadaya Tsuyukubo
+ * @since 1.10.0
+ */
+public enum GrpcObservation implements ObservationDocumentation {
```

Comment:
Since `DocumentedObservation` was renamed to `ObservationDocumentation`, should we rename `GrpcObservation` to to reflect that it is a documentation?

Reply:
Renamed to `GrpcObservationDocumentation` to align with existing other documentation classes.